server/app/helpers/request_helpers.rb

```diff
@@ -52,6 +52,18 @@ def halt_request(status, body = {})
     request.halt
   end
 
+  ##
```

Comment:
I think this load_grid thing could be done like the "require_current_user" and "validate_access_token" helpers, then you would always have a `current_grid` like you have `current_user`.

LGTM, just a thought.